src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
The logger is initialized with `OrderServiceImpl.class`, which might not be the most appropriate class for logging in the `Main` class. Consider using `Main.class` instead to reflect the correct context for logging.

Reply:
fixed